test/acceptance/tests/consul-dns/consul_dns_test.go

```diff
@@ -0,0 +1,79 @@
+package consuldns
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const podName = "dns-pod"
+
+func TestConsulDNS(t *testing.T) {
+	env := suite.Environment()
+	context := env.DefaultContext(t)
+	releaseName := helpers.RandomName()
+
+	cluster := framework.NewHelmCluster(t, nil, context, suite.Config(), releaseName)
+	cluster.Create(t)
+
+	k8sClient := context.KubernetesClient(t)
+	contextNamespace := context.KubectlOptions().Namespace
+
+	dnsService, err := k8sClient.CoreV1().Services(contextNamespace).Get(fmt.Sprintf("%s-%s", releaseName, "consul-dns"), metav1.GetOptions{})
+	require.NoError(t, err)
+
+	dnsIP := dnsService.Spec.ClusterIP
+
+	createdDNSPod(t, releaseName, context)
+
+	testDNS(t, context, dnsIP)
+}
+
+func createdDNSPod(t *testing.T, releaseName string, context framework.TestContext) {
```

Comment:
```suggestion
func createdDNSPod(t *testing.T, context framework.TestContext, releaseName string) {
```

Looks like we pass context first in `framework.NewHelmCluster()` so we could be consistent with that here. I haven't looked elsewhere so maybe this isn't always the pattern.